src/app/widgets/widget-gauge-ng-linear/widget-gauge-ng-linear.component.ts

```diff
@@ -252,7 +254,7 @@ export class WidgetGaugeNgLinearComponent extends BaseWidgetComponent implements
       needleSide: "both",
 
       highlights: [],
-      highlightsWidth: 0,
+      highlightsWidth: this.widgetProperties.config.gauge.highlightsWidth !== undefined && this.widgetProperties.config.gauge.highlightsWidth !== null ? this.widgetProperties.config.gauge.highlightsWidth : 5,
```

Comment:
Reworking initWidget() will remove the need to test highlightsWidth for undefined or null.